en/reference/validation-overrides.html

```diff
@@ -75,3 +73,113 @@ <h2 id="allow">allow</h2>
 </table>
+
+
+<h2 id="list">List of validation overrides</h2>
+
+<table>
+  <tr>
+    <th>ID</th>
+    <th>Required when...</th>
+    <th>Action needed or effect of change</th>
+  </tr>
+  <tr>
+    <td><code>indexing-change</code></td>
+    <td>Changing what tokens are expected and stored in field indexes.</td>
+    <td>Requires reindexing of data.</td>
+  </tr>
+  <tr>
+    <td><code>indexing-mode-change</code></td>
+    <td>Changing the index mode (streaming, indexed, store-only) of documents.</td>
+    <td>Requires reindexing of data.</td>
+  </tr>
+  <tr>
+    <td><code>field-type-change</code></td>
+    <td>Field type changes.</td>
+    <td>Requires refeeding data.</td>
+  </tr>
+  <tr>
+    <td><code>tensor-type-change</code></td>
+    <td>Tensor type change.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>resources-reduction</code></td>
+    <td>Large reductions in node resources (&gt; 50% of the current max total resources).</td>
+    <td>Might cause large load increase.</td>
+  </tr>
+  <tr>
+    <td><code>content-type-removal</code></td>
+    <td>Removal of a schema (causes deletion of all documents).</td>
+    <td>Causes loss of all documents in this schema.</td>
+  </tr>
+  <tr>
+    <td><code>content-cluster-removal</code></td>n
+    <td>Removal (or id change) of content clusters.</td>
+    <td>Causes loss of all documents in the cluster.</td>
+  </tr>
+  <tr>
+    <td><code>deployment-removal</code></td>
+    <td>Removal of production zones from deployment.xml.</td>
+    <td>Causes removal of all clustersand data in the zones.</td>
+  </tr>
+  <tr>
+    <td><code>global-document-change</code></td>
+    <td>Changing global attribute for document types in content clusters.</td>
+    <td>Requires stopping all nodes, applying validation override and starting nodes again.</td>
+  </tr>
+  <tr>
+    <td><code>global-endpoint-change</code></td>
+    <td>Changing global endpoints.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>zone-endpoint-change</code></td>
+    <td>Changing zone (possibly private) endpoint settings.</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>redundancy-one</code></td>
+    <td>The first deployment of an application with <code>redundancy=1</code> requires a validation override. A redundancy of 2 is required for clusters in production otherwise</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>paged-setting-removal</code></td>
+    <td>Typically removed due to <a href="https://docs.vespa.ai/en/attributes.html#paged-attributes-disadvantages">disadvantages</a> described in doc. May cause content nodes to run out of memory.</td>
+    <td>More data will be loaded into memory, might cause OOM.</td>
+  </tr>
+  <tr>
+    <td><code>certificate-removal</code></td>
+    <td>Removing data plane certificates, typically when moving to new certificates.</td>
+    <td>Unable to accesss endpoint with removed certificates.</td>
+  </tr>
+</table>
+
+
+<h3 id="list-internal">Validation overrides for internal use</h3>
```

Comment:
For end users, "internal use" is confusing. It might be easier to not expose this. To me, it is OK to keep the link to the source code, so easier to get to this

Reply:
Yeah, probably right. I'll use a link to source code instead. PR coming up